app/uk/gov/hmrc/merchandiseinbaggage/repositories/DeclarationJourneyRepository.scala

```diff
@@ -28,22 +28,23 @@ import uk.gov.hmrc.merchandiseinbaggage.model.core.DeclarationJourney.{format, i
 import uk.gov.hmrc.mongo.MongoComponent
 import uk.gov.hmrc.mongo.play.json.PlayMongoRepository
 
+import java.time.Instant
 import java.util.concurrent.TimeUnit
 import javax.inject.{Inject, Named, Singleton}
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 
 @Singleton
 class DeclarationJourneyRepository @Inject()(mongo: MongoComponent, @Named("declarationJourneyTimeToLiveInSeconds") ttlInSeconds: Int)
-    extends PlayMongoRepository[DeclarationJourney](
+    extends PlayMongoRepository[JsObject](
       mongo,
       "declarationJourney",
-      format,
+      implicitly[Format[JsObject]],
       Seq(
         IndexModel(ascending(id), IndexOptions().name("primaryKey").unique(true)),
         IndexModel(
-          ascending("createdAt"),
-          IndexOptions().name("timeToLive").unique(false).expireAfter(ttlInSeconds, TimeUnit.SECONDS)
+          ascending("createdAtDate"),
```

Comment:
please revert this index name we will need to replace indexes with a flag `replaceIndexes` will need to add it application.conf as well